include/RunAnalysis.h

```diff
@@ -14,12 +14,16 @@
 #include "llvm/Pass.h"
 #include "llvm/Support/SourceMgr.h"
 #include "llvm/Support/raw_ostream.h"
+
 #include <algorithm>
+#include <iostream>
 #include <iterator>
 #include <list>
 #include <map>
 #include <set>
+#include <sstream>
 #include <string>
+#include <vector>
```

Comment:
Vectors are used in the `splitString` function in `Utils.cpp` and `splitString` is used in `Annotation.cpp`. I prefer to put all C standard library directives used in the `RunAnalysis.h` since practically all the .h files include `RunAnalysis.h` , but whether it's better style to put `#include <vector>` in Utils.h I am unsure of either

Reply:
Can you add a comment explaining that these include directives are for all the .cpp files at the top of `RunAnalysis.h`? That would prevent my confusion in the future.